docs/reST/ref/math.rst

```diff
@@ -195,6 +195,36 @@ Multiple coordinates can be set using slices or swizzling
 
       .. ## Vector2.distance_squared_to ##
 
+   .. method:: move_toward
+
+      | :sl:`returns a vector that moves toward to the given vector at the given speed.`
```

Comment:
Everywhere you have written `speed` you mean `distance`. Speed is distance over time and you don't know how often this will be called.